clusters/hive/pools/openshift-ci/ci-ocp-4-6-0-amd64-aws-us-east-1_clusterpool.yaml

```diff
@@ -30,7 +30,7 @@ spec:
       region: us-east-1
   pullSecretRef:
     name: pull-secret
-  size: 1
+  size: 0
```

Comment:
With a size of zero, ClusterPool will react to a ClusterClaim by spinning up a fresh cluster... but that'll take ~40m to install. So unfortunately these zeros will render these pools unusable in CI.

...or ar we using this as a dirty way to flush out the few remaining consumers of these pools? :P

Reply:
Corrected.

I DO want to flush out the remaining consumers of old pools but it is not my call to make. :)